fms_extras/models/paged_llama.py

```diff
@@ -726,6 +728,22 @@ def factory(**kwargs):
 )
 models.register_model(_architecture_name, "70b", _llama_factory_factory(_70b_config))
 
+# llama3
+
+_8b_llama3_config = PagedLLaMAConfig(
+    src_vocab_size=128256,
+    emb_dim=4096,
+    norm_eps=1e-5,
+    nheads=32,
+    kvheads=8,
+    nlayers=32,
+    hidden_grow_factor=3.5,
+    multiple_of=1024,
+    max_expected_seq_len=8192,
+)
+
+models.register_model("paged_llama3", "8b", _llama_factory_factory((_8b_llama3_config)))
```

Comment:
I suspect `paged_llama3` isn't actually a different architecture vs `paged_llama`?

Reply:
yes, its the same architecture, just was trying to differentiate. I am ok with keeping it as `paged_llama` as well.

Reply:
IMO we should probably keep the same arch name when the architecture doesn't change

Reply:
Fixed in latest commit